addon.gypi

```diff
@@ -8,7 +8,7 @@
       '<(node_root_dir)/include/node',
       '<(node_root_dir)/src',
       '<(node_root_dir)/deps/uv/include',
-      '<(node_root_dir)/deps/v8/include'
+      '<(node_root_dir)/<(node_engine_include_dir)'
```

Comment:
Yeah, this is defined in `node-chakracore` [here](https://github.com/nodejs/node-chakracore/blob/xplat/common.gypi#L76-L78) and [here](https://github.com/nodejs/node-chakracore/blob/xplat/common.gypi#L92). I didn't bring in these changes when I created PR. I will move it from `node-chakracore` to `node-gyp` because that is the right place of those variables.